images/airflow/2.10.1/python/mwaa/config/airflow.py

```diff
@@ -242,6 +242,19 @@ def _get_opinionated_airflow_secrets_config() -> Dict[str, str]:
         "AIRFLOW__SECRETS__BACKEND_KWARGS": json.dumps(connection_lookup_pattern),
     }
 
+def _get_usage_data_config() -> Dict[str, str]:
+    """
+    Retrieve the environment variables for Airflow's usage data configuration section.
+
+    This config can be overridden by the user.
+
+    :returns A dictionary containing the environment variables.
+    """
+
+    return {
+        "AIRFLOW__USAGE_DATA_COLLECTION__ENABLED": "False",
```

Comment:
Is this key correct? Airflow env variables follow the convention: <section><double-underscore><sub-section><double-underscore><field-name-with-single-underscores>.

Reply:
Ashish, I think what you're getting at is the underscore pattern? This is expected here as this is the name of the section: usage_data_collection

https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/_api/airflow/providers/amazon/aws/sensors/emr/index.html